src/style.css

```diff
@@ -1 +1,132 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir-Book, sans-serif;
```

Comment:
The 'font-family' declaration should not include 'sans-serif' within the '@font-face' rule. The 'font-family' should only specify the custom font name, such as 'Avenir-Book'.